astropy/utils/diff.py

```diff
@@ -47,6 +47,18 @@ def diff_values(a, b, rtol=0.0, atol=0.0):
         return a != b
 
 
+def _ignore_terminal_size(func):
+    @functools.wraps(func)
+    def inner(*args, **kwargs):
+        from astropy import conf
+
+        with conf.set_temp("max_width", -1), conf.set_temp("max_lines", -1):
```

Comment:
This seems very `astropy.table.Table` specific. This function being in `utils` implies that it should be more generic than that? From the docstring:

```rst
Anything that can be turned into strings and compared using :py:mod:`difflib` should work.
```

Reply:
Well `Table` is really just the most likely class to benefit from this change, but it seems to me that the fix is generic. Isn't it ?